internal/csi/csi.go

```diff
@@ -50,18 +52,33 @@ func NewTester(cfg *ec2config.Config, terminateOnExit, journalctlLogs bool) (tes
 	return tester, nil
 }
 
-func CreateConfig(vpcID, branchOrPR string) *ec2config.Config {
-	cfg := ec2config.NewDefault()
+func CreateConfig(vpcID, prNum, githubAccount, githubBranch string) (cfg *ec2config.Config, err error) {
+	cfg = ec2config.NewDefault()
 	cfg.UploadTesterLogs = false
 	cfg.VPCID = vpcID
 	cfg.IngressRulesTCP = map[string]string{"22": "0.0.0.0/0"}
 	cfg.Plugins = []string{
 		"update-amazon-linux-2",
 		"install-go-1.11.3",
-		"install-csi-" + branchOrPR,
+	}
+	// If prNum is set to an non-empty string, the master branch of kubernetes-sigs/aws-ebs-csi-driver is used,
+	// regardless of whether or not githubAccount and githubBranch have different values.
+	if prNum != "" {
+		if githubAccount != "kubernetes-sigs" || githubBranch != "master" {
+			fmt.Printf("WARNING: PR number %s takes precedence over non-default GitHub account and/or branch.\n", prNum)
+		}
+		cfg.Plugins = append(cfg.Plugins, "install-csi-"+prNum)
+	} else {
+		cfg.CustomScript, err = createCustomScript(gitAccountAndBranch{
```

Comment:
Instead of having `createCustomScript()` and all the related code [below](https://github.com/aws/aws-k8s-tester/blob/668af90deddc7a5349f717ef55ac80327fa44cde/internal/csi/csi.go#L193-L241), should I use `createInstallGit()` from [plugins/plugins.go](https://github.com/aws/aws-k8s-tester/blob/master/ec2config/plugins/plugins.go#L380) like the pr number uses?

Reply:
`createCustomScript` is better, since this is only used inside `csi` package.

Reply:
ok!